linkerd/stack/tracing/src/lib.rs

```diff
@@ -193,6 +190,17 @@ where
     }
 }
 
+impl<T, G: GetSpan<T>, S> Drop for Instrument<T, G, S> {
+    fn drop(&mut self) {
+        let _span = self
+            .current_span
+            .take()
+            .unwrap_or_else(|| self.get_span())
+            .entered();
+        trace!("drop");
```

Comment:
nit, take it or leave it: if this used
```suggestion
        trace!(parent:  parent: self.current_span.as_ref(), "drop");
```
we wouldn't enter the span when the trace event isn't enabled. this might be a minor perf improvement when the span is at a higher level (e.g. `DEBUG`), and the drop event isn't enabled.

we can't easily create the if we do this, though, because we would need to hold it for longer than passing it to `parent:` --- otherwise, if it had just been created, it will be immediately dropped.

anyway, not really a blocker; current approach is fine!